Routes/CommerceRoutes.cs

```diff
@@ -70,6 +70,20 @@ public class CommerceRoutes : IRouteProvider {
                             {"area", "Nwazet.Commerce"}
                         },
                         new MvcRouteHandler())
+                },
+                new RouteDescriptor {

```

Comment:
Found the issue: it was in the javascript, where I would incorrectly use a token without checking whther it was not null.